src/package/external/external.csproj

```diff
@@ -57,6 +57,10 @@
       <Version>15.5.0-preview-1050541</Version>
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.Internal.Intellitrace">
```

Comment:
Require changes in sign.proj since new assemblies are now packaged.

Reply:
Done.